tools/monorepo-utils/src/md-docs/lib/generate-manifest.ts

```diff
@@ -34,6 +34,7 @@ async function processDirectory(
 	projectName: string,
 	baseUrl: string,
 	baseEditUrl: string,
+	fullPathToDocs: string,
```

Comment:
We have to keep track of the full path because this method is called recursively. We use this to generate a relative path to the file being processed when making an ID.